### PR TITLE
Remove serializable from API

### PR DESCRIPTION
**Pull request by see-quick**
### Type of change

- Enhancement / new feature
- Refactoring

### Description

This PR addresses an issue [1]. TLDR; Remove the `Serializable` interface because It's never serialized using Java serialization. 

[1] - https://github.com/strimzi/strimzi-kafka-operator/issues/2075

### Checklist

- [x] Make sure all tests pass
- [x] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Reference relevant issue(s) and close them after merging

### CODE REVIEW

**Review by Frawless — APPROVED** [review]
I am not SME, but LGTM as long as everything works

File: api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java [hunk 1]
```diff
@@ -23,7 +23,6 @@
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 public class KafkaBridgeAdminClientSpec extends KafkaBridgeClientSpec {
-    private static final long serialVersionUID = 1L;
```
Comment (scholzj) [inline api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java:26]: As I said before - please stop creating the empty lines at the top of the file. This applies to most of the files here.

**Review by scholzj — CHANGES_REQUESTED** [review]

File: api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java (hunk 1 above)
Comment (Frawless) [reply api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java:26]: Guess it is something what could be added to checkstyle?

Comment (see-quick) [reply api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java]: Yeah, sorry I forgot about that :/ updated 👍

Comment (see-quick) [reply api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java]: Also there is another problem that our checkstyle ignores more lines between import groups f.e., 
```

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import io.strimzi.api.kafka.model.common.Constants;
import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
import io.strimzi.crdgenerator.annotations.Description;
import io.strimzi.crdgenerator.annotations.DescriptionFile;
import io.strimzi.crdgenerator.annotations.Minimum;
import io.sundr.builder.annotations.Buildable;
import lombok.EqualsAndHashCode;
import lombok.ToString;


import java.util.HashMap;
import java.util.Map;
```
this treats as okay. Which is not good FPOV. I would update check style :
```diff
- <property name="separateLineBetweenGroups" value="false"/>
+ <property name="separateLineBetweenGroups" value="true"/>
```

**Comment by scholzj** [comment]
I think you committed here way more than you wanted ...

**Review by scholzj — APPROVED** [review]

**Pull request merged**